src/coreComponents/physicsSolvers/inducedSeismicity/inducedSeismicityFields.hpp

```diff
@@ -0,0 +1,176 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file inducedSeismicityFields.hpp
+ */
+
+#ifndef GEOS_PHYSICSSOLVERS_INDUCEDSEISMICITY_INDUCEDSEISMICITYFIELDS_HPP_
+#define GEOS_PHYSICSSOLVERS_INDUCEDSEISMICITY_INDUCEDSEISMICITYFIELDS_HPP_
+
+#include "common/DataLayouts.hpp"
+#include "mesh/MeshFields.hpp"
+
+namespace geos
+{
+
+namespace fields
+{
+
+namespace inducedSeismicity
```

Comment:
Perhaps all of these fields are required, but it is worth asking if any can be dropped and just recomputed as needed.

Reply:
Deleted defunct fields and one of the _n variables that is now updated on the fly